ibis/formats/pandas.py

```diff
@@ -21,23 +16,19 @@
 from ibis.formats.pyarrow import PyArrowData, PyArrowSchema, PyArrowType
 
 if TYPE_CHECKING:
+    import pandas as pd
     import polars as pl
     import pyarrow as pa
 
-_has_arrow_dtype = hasattr(pd, "ArrowDtype")
-
-if not _has_arrow_dtype:
-    warnings.warn(
-        f"The `ArrowDtype` class is not available in pandas {pd.__version__}. "
-        "Install pandas >= 1.5.0 for interop with pandas and arrow dtype support"
-    )
-
 geospatial_supported = _find_spec("geopandas") is not None
 
 
 class PandasType(NumpyType):
     @classmethod
     def to_ibis(cls, typ, nullable=True):
+        import pandas as pd
```

Comment:
Importing pandas globally in this file would be reasonable so that `ibis.formats.pandas` actually depends on pandas. Same applies to `ibis.formats.numpy`.

Reply:
Yep, thanks for the poke! I was able to move these imports up without breaking anything and still allowing no pyarrow/pandas/numpy installation.